packages/wonder-blocks-cell/src/components/__docs__/basic-cell.stories.js

```diff
@@ -0,0 +1,204 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+
+import type {StoryComponentType} from "@storybook/react";
+
+import type {IconAsset} from "@khanacademy/wonder-blocks-icon";
+
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+import BasicCellArgTypes from "./basic-cell.argtypes.js";
+
+import BasicCell from "../basic-cell.js";
+
+export default {
+    title: "Layout / Cell / Basic Cell",
+    component: BasicCell,
+    argTypes: BasicCellArgTypes,
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
```

Comment:
This is used to include a link to GH and show the current package version.
<img width="1046" alt="Screen Shot 2021-12-21 at 6 47 28 PM" src="https://user-images.githubusercontent.com/843075/147011914-61d97117-d0ba-4988-a300-41380d4f7d17.png">